test_shopinvader_payment/tests/test_payment.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
```

Comment:
@sebastienbeau This file is a new one... The copyright year should be 2019. The authors should put into the readme.The coding can be removed.
This test file is used to test the shopinvader manuel payment.. Its name should be `test_shopinvader_payment_manual` and the classname `TestShopinvaderPaymentManual()`